pulpcore/tests/functional/api/using_plugin/test_repo_versions.py

```diff
@@ -110,13 +110,13 @@ def test_02_sync_content(self):
 
         self.assertIsNotNone(repo['_latest_version_href'])
 
-        content = get_content(repo)
+        content = get_file_content(repo)
```

Comment:
As mentioned earlier, we could just always do ```get_content(repo)['file']``` instead.  Open to changing this if QE would prefer it that way.